docs/reference/modules/snapshots.asciidoc

```diff
@@ -44,9 +44,9 @@ If you register same snapshot repository with multiple clusters, only
 one cluster should have write access to the repository. All other clusters
 connected to that repository should set the repository to `readonly` mode.
 
-NOTE: The snapshot format can change across major versions, so if you have
+IMPORTANT: The snapshot format can change across major versions, so if you have
 clusters on different major versions trying to write the same repository,
-new snapshots written by one version will not be visible to the other. While
+snapshots written by one version may not be visible to the other. While
```

Comment:
I wonder if we need to be harsher here and say that writing to the repository from an older version once it's been written to by a new version can lead to repository corruption.

Reply:
I agree